README.md

```diff
@@ -365,6 +366,38 @@ The maxLength attribute added to the input. Specifies the maximum number of char
     ...>
 ```
 
+<a name="inline"></a>
+##### inline (optional)
+The inline attributes decides whether the input fields and selected tags will be rendered in-line.
+
+```
+<ReactTags
+    inline = {false}
+    ...>
+```
+
+If inline is true (default value)
+
+```
+<div className="...">
+	{tagItems}
```

Comment:
This won't be needed after adding the screenshot :)
Can you do it the below way
```
<ReactTags
  inline
>
```
add the screenshot when inline true

```
<ReactTags
  inline={false}
>
```
Add the screenshot for the inline false